config/prod/cmolitor-data_tasks-ec2.yaml

```diff
@@ -0,0 +1,47 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2.yaml
+stack_name: cmolitor-data-tasks-ec2
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Data Curation"
```

Comment:
we prefer no spaces.  makes it easier to query resources and track them.

Reply:
Changed it.